docs/sources/reference/components/loki.source.podlogs.md

```diff
@@ -8,13 +8,13 @@ title: loki.source.podlogs
 
 `loki.source.podlogs` discovers `PodLogs` resources on Kubernetes and, using
 the Kubernetes API, tails logs from Kubernetes containers of Pods specified by
-the discovered them.
+the discovered Pods.
```

Comment:
Is this correct? The sentence was incomplete....

Reply:
I think I meant to say:

> `loki.source.podlogs` discovers `PodLogs` resources on Kubernetes and, using the Kubernetes API, tails logs from Kubernetes containers of Pods specified by the PodLogs resource. 

Basically: 

* `loki.source.podlogs` discovers PodLogs resources 
* PodLogs resources give rules for which Pods to discover on your cluster 

`loki.source.podlogs` _indirectly_ discovers Pods, but only based on the rules in the directly discovered PodLogs resources.

Reply:
Updated the intro text as we discussed.